.github/reusable_workflows/spell_check_pr.yml

```
@@ -0,0 +1,22 @@
+name: Check spelling
```

Is this better? :thinking: 

```suggestion
name: Check spelling for PR
```

---

Or, remove `pr` from the file name.